src/test/java/org/folio/service/finance/FiscalYearServiceTest.java

```diff
@@ -64,12 +83,12 @@ void testShouldReturnCurrentFiscalYearForLedger() {
   }
 
   @Test
-  void testShouldThrowHttpException() {
-
+  void testShouldThrowHttpException() throws IllegalAccessException, NoSuchFieldException {
+    FiscalYear sampleFiscalYear = new FiscalYear();
+    FiscalYearService fiscalYearService = new FiscalYearService(restClientMock, fundServiceMock);
```

Comment:
you use @InjectMocks annotation, so you should not create instance of FiscalYearService, see other tests for example